# conversion between vrpn asset names and UWB identifiers
# UWB identifiers should match the anchor_address/source_address
# identifiers from the UWB range messages
# Format is: vrpn_asset_name: "UWB identifier"

uwb_mount_1: "000000000000141D"
uwb_mount_2: "0000000000001639"
uwb_mount_3: "0000000000005950"
uwb_mount_4: "0000000000001B20"
uwb_mount_5: "000000000000198C"

# rate (in Hz) at which vrpn messages are published
# This rate is used for approximate time synchronization of all
# vrpn asset pose publications
vrpn_rate_hz: 120

# vrpn topic prefix
# prefix is used to find all vrpn messages
# vrpn
vrpn_prefix: "vrpn_client_node"
